README.rst

```diff
@@ -63,8 +64,8 @@ Imagine you have an application that has one dependency:
   $ cat requirements.txt
   tensorflow
 
-
-Tool provided by this project will tell you how dependencies could be resolved:
```

Comment:
This project provides a tool which tells you what dependencies...

Reply:
I think this sentence was removed.